setup.py

```diff
@@ -88,6 +88,13 @@ def _scikit_umfpack_version(pkg_name):
     except:
         return None
 
+
+def _primme_version(pkg_name):
+    import primme
+
+    return '3.0.0'
```

Comment:
OK. Most versions in `sfepy/version.py` are too old anyway, so maybe it is time to revisit or ditch those checks?